packages/account/src/Components/form-body/form-body.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+// [TODO] - To be removed once CFD is configured to use the new form-body component
+import React, { PropsWithChildren, Fragment } from 'react';
```

Comment:
Do you still need to import React?

Reply:
Yes. I was getting an error in a component of CFD package that was using this. When I tried adding the config to whole deriv-app to use new jsx transformer, there were TS issues. So I added comment